samples/Facebook/README.md

```diff
@@ -11,20 +11,22 @@
     ```
 
 4. Use CLI tool `paconn` to create a new custom connector.
-5. On Facebook developer dashboard update 'Valid OAuth Redirect URIs' under Products >> Facebook Login >> Settings.
-  You can find the redirect URI on security page of your custom connector.
+5. On Facebook developer dashboard update 'Valid OAuth Redirect URIs' under Products >> Facebook Login >> Settings. You can find the redirect URI on security page of your custom connector.
 
-## Supported Actions
+## Supported Triggers
 
-As part of this sample following actions are supported:
+As part of this sample following triggers are supported:
+
+* `When there is a new post on my timeline`: Triggers a new flow when there is a new post on your timeline. It is possible for this trigger to not activate for all posts since detection of a post must pass several privacy checks including a person's privacy settings on Facebook.
 
-* `Get my Pages`: Lists all the pages that you own.
+## Supported Actions
 
-* `Get feeds from my timeline`: Gets feed of posts from your timeline.
+As part of this sample following actions are supported:
 
-* `Get page access token`: Get a page access token for a page you own.
+* `Get my feed`: Gets the feed from your timeline.
 
-* `Get Page Feeds`: Gets feed of posts from a page
+* `Get page feed`: Get posts from the feed of a specified page.
```

Comment:
Please follow consistent pattern. This should be `gets` just like the previous line, or fix the previous line.